helm_deploy/values-dev.yaml

```diff
@@ -18,8 +18,8 @@ generic-service:
     PRISONER_SEARCH_API_URL: "https://prisoner-offender-search-dev.prison.service.justice.gov.uk"
     # PROBATION_SEARCH_API_URL: "https://probation-offender-search-dev.hmpps.service.justice.gov.uk"
     PRISON_REGISTER_API_URL: "https://prison-register-dev.hmpps.service.justice.gov.uk"
-    EXIT_SURVEY_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
-    PHASE_BANNER_LINK: "https://eu.surveymonkey.com/r/Preview/?sm=CQIWAIrnp68aIE1_2BNn3aRSTf_2BXWLpL8B9mtgPre1TKl4OznrnkajcbEz6SvRvXwn"
+    EXIT_SURVEY_LINK: "https://eu.surveymonkey.com/r/V9JSHL3"
```

Comment:
Maybe just leave the dev one to the preview? Dont want to be sending feedback from dev!

Reply:
I think being able to see how the survey looks in dev is useful, as this is where our people have access. We should probably collect an email address in the survey so we can differentiate real from test responses? Not sure on this. I did ask in slack and Ana was not bothered what showed in dev whilst Adam said make them all the same.